airflow/contrib/operators/gcp_bigtable_operator.py

```diff
@@ -0,0 +1,415 @@
+
+    :type project_id: str
+    :param project_id: The ID of the GCP project.
+    :type instance_id: str
+    :param instance_id: The ID of the Cloud Bigtable instance that will hold the new table.
+    :type table_id: str
+    :param table_id: The ID of the table to be created.
+    """
+    REQUIRED_ATTRIBUTES = ('project_id', 'instance_id', 'table_id')
+    template_fields = ['project_id', 'instance_id', 'table_id']
+
+    @apply_defaults
+    def __init__(self,
+                 project_id,
+                 instance_id,
+                 table_id,
+                 initial_split_keys=None,
+                 column_families=None,
+                 *args, **kwargs):
+        self.project_id = project_id
+        self.instance_id = instance_id
+        self.table_id = table_id
+        self.initial_split_keys = initial_split_keys or list()
+        self.column_families = column_families or dict()
+        self._validate_inputs()
+        self.hook = BigtableHook()
+        self.instance = None
+        super(BigtableTableCreateOperator, self).__init__(*args, **kwargs)
+
+    def _compare_column_families(self):
+        table_column_families = self.hook.get_column_families_for_table(self.instance, self.table_id)
+        if set(table_column_families.keys()) != set(self.column_families.keys()):
+            self.log.error("Table '%s' has different set of Column Families", self.table_id)
+            self.log.error("Expected: %s", self.column_families.keys())
+            self.log.error("Actual: %s", table_column_families.keys())
+            return False
+
+        for key in table_column_families.keys():
+            # there is difference in structure between local Column Families and remote ones
+            # the local ones are kept in `gc_rule` property of remote ones.
+            if table_column_families[key].gc_rule != self.column_families[key]:
+                self.log.error("Column Family '%s' differs for table '%s'.", key, self.table_id)
+                return False
+        return True
+
+    def execute(self, context):
+        self.instance = self.hook.get_instance(self.project_id, self.instance_id)
+        if not self.instance:
+            raise AirflowException("Dependency: instance '{}' does not exist in project '{}'.".format(
+                self.instance_id, self.project_id))
+        try:
+            self.hook.create_table(
+                self.instance,
+                self.table_id,
+                self.initial_split_keys,
+                self.column_families
+            )
+        except google.api_core.exceptions.AlreadyExists:
+            if not self._compare_column_families():
+                raise AirflowException(
+                    "Table '{}' already exists with different Column Families.".format(self.table_id))
+            self.log.info("The table '{}' already exists. Consider it as created", self.table_id)
```

Comment:
There is a problem here with '{}' -> "%s". It causes errors during unit tests@ travis: https://travis-ci.org/PolideaInternal/incubator-airflow/jobs/470924738#L4471